phases/19-capstone-projects/43-hdf5-tokenized-corpus/code/main.py

```
@@ -0,0 +1,538 @@
+            raise ValueError("at least one shard entry is required")
+        self._entries = shard_entries
+        self._files: list[h5py.File] = []
+        try:
+            for entry in shard_entries:
+                self._files.append(h5py.File(entry.path, "r", swmr=True))
+        except Exception:
+            for opened in self._files:
+                try:
+                    opened.close()
+                except Exception:
+                    pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)